en/_posts/2016-07-04-local-stateful.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title:  "Local stateful"
+categories: docker
+author: tgermain
+lang: en
+---
+
+
+
+This tutorial explains how local stateful and volumes work.
```

Comment:
What do you mean by "local stateful and volumes" ? Is it "Locally stateful containers and volumes ?"